core/src/data-generator/prepared-dataset.ts

```diff
@@ -86,6 +94,9 @@ export interface PreparedDatasetProps {
  * 16000000600  , s3://<path>/<to>/<folder>/time_range_start=16000000600/file2.csv
  *
  * 16000000600  , s3://<path>/<to>/<folder>/time_range_start=16000000600/whichever-file....csv
+ * 
```

Comment:
What happen if pass both of them? Should we check if both are passed?

Reply:
now throws an error if both are passed